pages/en/developer/create-subgraph-hosted.mdx

```diff
@@ -27,7 +27,7 @@ The Graph Network supports subgraphs indexing mainnet Ethereum:
 - `matic`
 - `mumbai`
 - `fantom`
-- `bnb`
+- `bsc`
```

Comment:
Although the network identifier needs to still be `bsc`, the chain was indeed renamed to `BNB`, do we want to add this fact in there, maybe like suggested edit.
```suggestion
- `bsc` (now known as BNB chain)
```

Or maybe we are able to update deployment to `bnb` and have an alias?

Reply:
I guess the same case can be made for xdai and matic. But a comment can definitely helpful, especially for people who entered the space recently